kagami/core/models/worker_info.py

```diff
@@ -1,10 +1,23 @@
 import logging
+from enum import Enum
 
 from .provider_info import ProviderInfo
 
 logger = logging.getLogger(__name__)
 
 
+class WorkerStatus(Enum):
+    ONLINE = "online"
+    OFFLINE = "offline"
+    MAINTENANCE = "maintenance"
+    ERROR = "error"
+
+
+class WorkerVerifyStatus(Enum):
+    VERIFIED = "Verified"
+    UNVERIFIED = "Unverified"
```

Comment:
This part is already define in database of worker and only used in back_end. Interface to front-end doesn't need it

Reply:
> This part is already defined in database of worker and only used in back_end. Interface to front-end doesn't need it

Received. I'll censor all the codes before the next commit.